### ISSUE TITLE
Feature request: Indicate why you see followed-tag posts in timeline

### ISSUE THREAD

**Issue by henrik** [Dimillian/IceCubesApp#1264]
One of the few things I miss from Ivory is how it shows that you’re seeing a post in your timeline because you follow the tag.

Can be a little confusing sometimes to see a post in your timeline from someone you don’t follow and which is not a boost.

This is what it looks like in Ivory:

![8DD39416-7FD7-4FA6-8233-9857B60054ED](https://user-images.githubusercontent.com/216/226071217-47429591-1641-42ff-99d6-91c279c85a29.png)

**Comment by sothawo** [comment]
Optional setting please, if implemented. It takes away available screen space. And, if I see a post from someone don't follow and that was not boosted, it must be a hashtag anyway I think.

**Comment by henrik** [comment]
That’s fair - I would of course not mind it being an option.

Even though it technically must be a followed hashtag, I see a few benefits:

- I got confused by this as a newer Mastodon user. I followed some hashtag, then forgot. Saw stuff in my timeline I did not expect. Did not have a clear mental model of exactly what shows up in your timeline.
- Sometimes you’re not in your home timeline but the federated list or a remote local. Then you see unfollowed posts for other reasons. This makes it clearer at a glance. 
- If you don’t remember every hashtag you follow (maybe you follow one that gets posts rarely) this makes it immediately clear what’s going on.
